### PR TITLE
work on RFC: add constant pi divided by four #76

### LINKED ISSUE #76: RFC: add constant pi divided by four
## Checklist

> Please ensure the following tasks are completed before filing an issue.

* [x] Read and understood the [Code of Conduct][code-of-conduct].
* [x] Searched for existing issues and pull requests.
* [x] If this is a general question, searched the [FAQ][faq] for an existing answer.
* [x] If this is a feature request, the issue name begins with `RFC: `.

## Description

> Description of the issue (or feature request).

Along lines similar to `@stdlib/math/constants/float64-half-pi`, this RFC proposes the addition of a new constant package to `@stdlib/math/constants`; namely, `pi/4`. The constant `pi/4` finds common occurrence in various math implementations; e.g., `acos`, `asin`, `atan`.

Package name possibilities include:

```
@stdlib/math/constants/float64-pi-over-four
@stdlib/math/constants/float64-quarter-pi
@stdlib/math/constants/float64-pi-div-four
@stdlib/math/constants/float64-pio4
@stdlib/math/constants/float64-fourth-pi
```

The last name possibility is consistent with `float64-half-pi`, but is a bit more awkward/less commonly said. Individuals may be more likely to say "pi over four" than "fourth pi". But then again, individuals may be more likely to say "pi over two" rather than "half pi". In terms of conventions, a separate convention for similar type constants could be imposed where the divisor is greater than `2`; i.e., keep `*-half-pi` type names, but then also have `*-pi-over-three`, `*-pi-over-four`, etc.

FWIW, Boost follows the convention `*-fourth-pi`.

## Related Issues

> Does this issue (or feature request) have any related issues?

Acceptance of this proposal would impact PR #75.

## Questions

> Any questions for reviewers?

* According to what naming convention should the proposed package follow?

[code-of-conduct]: https://github.com/stdlib-js/stdlib/blob/master/CODE_OF_CONDUCT.md
[faq]: https://github.com/stdlib-js/stdlib/blob/master/FAQ.md

### PR DESCRIPTION
**Pull request by chrisdamba**
Resolves #76 .

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

* [x] Read, understood, and followed the [contributing guidelines][contributing], including the relevant style guides.
* [x] Read and understand the [Code of Conduct][code-of-conduct].
* [x] Read and understood the [licensing terms][license].
* [x] Searched for existing issues and pull requests __before__ submitting this pull request.
* [x] Filed an issue (or an issue already existed) __prior to__ submitting this pull request.
* [x] Rebased onto latest `develop`.
* [x] Submitted against `develop` branch.

## Description

> What is the purpose of this pull request?

This pull request:

* is work on RFC: add constant pi divided by four #76

## Related Issues

> Does this pull request have any related issues?

This pull request:

* resolves #76

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

---

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/master/CONTRIBUTING.md
[code-of-conduct]: https://github.com/stdlib-js/stdlib/blob/master/CODE_OF_CONDUCT.md
[license]: https://github.com/stdlib-js/stdlib/blob/master/LICENSE

### CODE REVIEW

File: lib/node_modules/@stdlib/math/constants/float64-fourth-pi/lib/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+/**
+* One fourth times the mathematical constant `π`.
+*
+* @module @stdlib/math/constants/float64-half-pi
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/constants/float64-fourth-pi/lib/index.js:6]: Need to fix the pkg path.

> @stdlib/math/constants/float64-fourth-pi

File: lib/node_modules/@stdlib/math/constants/float64-fourth-pi/lib/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* @type {number}
+*
+* @example
+* var FOURTH_PI = require( '@stdlib/math/constants/float64-half-pi' );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/constants/float64-fourth-pi/lib/index.js:10]: Need to fix the require path.

> @stdlib/math/constants/float64-fourth-pi

File: lib/node_modules/@stdlib/math/constants/float64-fourth-pi/lib/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* // returns 7.85398163397448309616e-1
+*/
+
+
+// MAIN //
+
+/**
+* One half times the mathematical constant `π`.
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/constants/float64-fourth-pi/lib/index.js:18]: > One fourth times the...

File: lib/node_modules/@stdlib/math/constants/float64-fourth-pi/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Usage
+
+``` javascript
+var FOURTH_PI = require( '@stdlib/math/constants/float64-fourth-pi' );
+```
+
+#### FOURTH_PI
+
+One half times the mathematical constant [π][pi].
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/constants/float64-fourth-pi/README.md:15]: > One fourth times the...

**Review by kgryte — CHANGES_REQUESTED** [review]

**Comment by kgryte** [comment]
Thanks for the PR! After making the requested changes, see the [Git guide](https://github.com/stdlib-js/stdlib/tree/develop/docs/style-guides/git) for `squash` and commit message info. When you `squash`, you should be able to amend the message per the guide.

**Comment by chrisdamba** [comment]
Thanks @kgryte for the guidance. I apologise for the mess, will get it right eventually :)

**Comment by kgryte** [comment]
@chrisdamba No worries! This is a valuable experience for us in knowing how to best onboard people into the project. Obviously, we have a fair amount of work to do to make the process smooth and sufficiently informed so that contributors can get up to speed as fast as possible.

**Comment by chrisdamba** [comment]
Tidying up the PR by deleting and submitting a new one with the requested changes and clean commit based on the guidance.

**Pull request closed without merging**